Marker/Classes/TextStyle.swift

```diff
@@ -11,97 +11,75 @@ import UIKit
 /**
  *  Encapsulates style information to be applied when displaying text.
  */
-public protocol TextStyle {
-    
-    /// Text color.
-    var textColor: UIColor? { get }
+public struct TextStyle {
 
     /// Font for displaying regular text.
-    var font: UIFont { get }
+    public var font: UIFont
```

Comment:
is there a reason you need to specify the public here? the struct TextStyle is already public